### PR TITLE
Add call to contribute to bottom of issue templates

### PR DESCRIPTION
**Pull request by samajammin**
## Description

Add call to contribute to bottom of issue templates

## Related Issue
None

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -42,3 +42,7 @@ Steps to reproduce the behavior:
 **Additional context**
 
 <!-- Add any other context about the problem here. -->
+
+**Want to contribute?**
+
+We love contributions from the Ethereum community! Comment on this issue below if you're interested in helping out with a PR.
```
Comment (ryancreatescopy) [inline .github/ISSUE_TEMPLATE/bug_report.md:48]: Not clear to me what the "issue below" would be? Do you mean they should comment on the issue they're creating to signal interest?

If so, maybe something like

"We love contributions from the Ethereum community! Remember to comment on your issue if you're interested in helping out with a PR".

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (samajammin) [reply .github/ISSUE_TEMPLATE/bug_report.md:48]: Ya @ryancreatescopy this came from a conversation in retro last week. Main goal here is to have this prompt not for the creator of the issue but for folks who come across an open issue.

Comment (samajammin) [inline .github/ISSUE_TEMPLATE/bug_report.md:48]:
```suggestion
We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -21,3 +21,7 @@ assignees: ""
 **Additional context**
 
 <!-- Add any other context or screenshots about the feature request here. -->
+
+**Want to contribute?**
+
+We love contributions from the Ethereum community! Comment on this issue below if you're interested in helping out with a PR.
```
Comment (samajammin) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]:
```suggestion
We love contributions from the Ethereum community! Please comment on an issue if you're interested in helping out with a PR.
```

**Review by wackerow — APPROVED** [review]
Looks good, can always adjust

**Pull request merged**